### PR TITLE
Update travis

### LINKED ISSUE #9: compile error
hello 
image

last rev azerothcore

### PR DESCRIPTION
**Pull request by talamortis**
Updated to see if travis will compile

Closes https://github.com/azerothcore/mod-individual-xp/issues/9

### CODE REVIEW

File: src/Individual_XP.cpp [hunk 1]
```diff
@@ -119,7 +119,6 @@ class Individual_XP_command : public CommandScript
             };
 
             return IndividualXPBaseTable;
-        }
```
Comment (BarbzYHOOL) [inline src/Individual_XP.cpp:122]: error?

**Comment by talamortis** [comment]
Yeah, I'll look at it when I got more time

**Comment by BarbzYHOOL** [comment]
i don't know, i feel like the testers didn't properly test

So I enabled travis to see if it compiles there but not sure how to make it show up in the PR

**Comment by BarbzYHOOL** [comment]
lol the build completely failed

**Comment by BarbzYHOOL** [comment]
https://github.com/azerothcore/skeleton-module/blob/master/.travis.yml

**Comment by talamortis** [comment]
yeah ill fix when i get more time

**Review by pangolp — APPROVED** [review]

**Comment by pangolp** [comment]
What would be missing to finalize this pull request?

**Comment by BarbzYHOOL** [comment]
need to update the travis file, see if it compiles and then fix ?

have you tested it @pangolp ?

**Comment by wizzymore** [comment]
https://github.com/azerothcore/mod-individual-xp/pull/12 should fix the travis problems and help you better debug the module

File: src/Individual_XP.cpp (hunk 1 above)
Comment (wizzymore) [reply src/Individual_XP.cpp:122]: This needs to be re-added

**Comment by pangolp** [comment]
What is the problem that generates in travis and how can I test it?

**Comment by wizzymore** [comment]
> What is the problem that generates in travis and how can I test it?

There is no problem with travis module-specific, is just it is not correctly set-up.

As for this pr, this pr is resolving a linux only error:
```diff
-void OnGiveXP(Player* p, uint32& amount, Unit* victim) override
+void OnGiveXP(Player* p, uint32& amount, Unit* /*victim*/) override
```
Doing this change might make linux compiling of the module work, but can't know this as travis doesn't currently work and no one tested the real change

**Comment by wizzymore** [comment]
Hi @talamortis, thank you so much for your contribution.

You can find the complete pull request that fixed travis here: https://github.com/azerothcore/mod-individual-xp/pull/14

**Pull request closed without merging**